project.inlang.json

```diff
@@ -1,15 +1,26 @@
 {
-	"$schema": "https://inlang.com/schema/project-settings",
-	"sourceLanguageTag": "en",
-	"languageTags": ["de", "en", "zh", "sk"],
-	"modules": [
-		"https://cdn.jsdelivr.net/npm/@inlang/plugin-json@4/dist/index.js",
-		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation/dist/index.js",
-		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern/dist/index.js",
-		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source/dist/index.js",
-		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-identical-pattern/dist/index.js"
-	],
-	"plugin.inlang.json": {
-		"pathPattern": "./inlang/source-code/website/lang/{languageTag}.json"
-	}
-}
+  "$schema": "https://inlang.com/schema/project-settings",
+  "sourceLanguageTag": "en",
+  "languageTags": [
+    "en",
+    "de",
+    "pt-BR",
+    "sk",
+    "zh"
+  ],
+  "modules": [
+    "https://cdn.jsdelivr.net/npm/@inlang/plugin-json@4/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source/dist/index.js",
+    "https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-identical-pattern/dist/index.js"
+  ],
+  "plugin.inlang.json": {
+    "pathPattern": "./inlang/source-code/website/lang/{languageTag}.json",
+    "ignore": [],
```

Comment:
@BRGustavoRibeiro, thanks for your contribution :) 
Could you let me know why you changed the settings? Or did you use the editor, and it changed it?

Reply:
I only used the web editor - it probably changed it automatically.

Reply:
Oh, sorry, I also encountered this bug myself and forgot to tell you :D It's a 'bug' on the Editor side. 

by the way, I have read the thread about your app. If you need a German translation, please feel free to ping me any time. :)